packages/chat/src/ChatClient.ts

```diff
@@ -2139,15 +2140,21 @@ export class ChatClient extends IrcClient {
 		if (attributes.replyTo) {
 			tags['reply-parent-msg-id'] = extractMessageId(attributes.replyTo);
 		}
-		await this._messageRateLimiter.request(
-			{
-				type: 'say',
-				channel: toChannelName(channel),
-				text,
-				tags
-			},
-			rateLimiterOptions
+
+		// eslint-disable-next-line @typescript-eslint/promise-function-async
```

Comment:
Please don't do this. Using async/await even in the callbacks will provide better stack traces.